man/man1/z.md

```diff
@@ -60,9 +60,6 @@ For more details about frecency, see https://github.com/rupa/z.
 
 ## CONFIGURATION
 
-  * `set -U Z_CMD "j"`:
-    Change commands to `j` and `jo`
-
```

Comment:
Maybe explain somewhere that aliases can be made (preferably) with `abbr` or `function --wraps`

Reply:
Yeah that sounds good. I don't really have the time anymore to work on this though.